lmctl/client/models/intents.py

```diff
@@ -94,6 +94,51 @@ def __eq__(self, other):
         return (isinstance(other, DeleteAssemblyIntent) and self.assembly_name==other.assembly_name
             and self.assembly_id==other.assembly_id)
 
+class RetryAssemblyIntent(Intent):
+
+    def __init__(self, process_id: str = None):
```

Comment:
As `process_id` is the only argument to be provided I think it should not default to `None` and we should validate it has been set in the constructor, raising a `ValueError` if not

Reply:
Addressed.

Reply:
`process_id = " "` would be allow to pass here. We'll need to first strip whitespace `if process_id is None or len(process_id.strip()) == 0:`

Reply:
Addressed.